travis.sh

```diff
@@ -282,8 +284,18 @@ function build_workspace() {
    # Console output fix for: "WARNING: Could not encode unicode characters"
    export PYTHONIOENCODING=UTF-8
 
-   # For a command that doesn’t produce output for more than 10 minutes, prefix it with travis_run_wait
-   travis_run_wait 60 --title "catkin build" catkin build --no-status --summarize
+   # Set compile flag to improve ccache hit rate
+   export CXXFLAGS="$CXXFLAGS -fdebug-prefix-map=$(pwd)=."
+   echo "CXXFLAGS = $CXXFLAGS"
+
+   # If test whitelist is set, explicitly build that project
+   if [ "${TEST_WHITELIST:-}" ]; then
```

Comment:
Why so complicated? If `TEST_WHITELIST` is empty, the if-branch command equals the else-branch one.

Reply:
Good point, I didn't think of that.